src/main/java/Duke.java

```diff
@@ -1,10 +1,96 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
 public class Duke {
+    final private static String BOT_NAME = "Python";
+    final private static int HORIZONTAL_LINE_LENGTH = 80;
+
+    final private static String pythonAsciiArt =
+                    "\t ____        _   _                 \n" +
+                    "\t|  _ \\ _   _| |_| |__   ___  _ __  \n" +
+                    "\t| |_) | | | | __|  _ \\ / _ \\|  _ \\ \n" +
+                    "\t|  __/| |_| | |_| | | | (_) | | | |\n" +
+                    "\t|_|    \\__, |\\__|_| |_|\\___/|_| |_|\n" +
+                    "\t       |___/                        ";
+    final private static String pythonEmoji = "\uD83D\uDC0D";
```

Comment:
```suggestion
    final private static String PYTHONEMOJI = "\uD83D\uDC0D";
```

Constants are recommended to be in ALL CAPS

Reply:
Changed! Thank you